test/MUIDataTable.test.js

```diff
@@ -1179,7 +1179,10 @@ describe('<MUIDataTable />', function() {
     instance.selectRowUpdate('cell', { index: 1, dataIndex: 1 });
     shallowWrapper.update();
 
-    const expectedResult = [{ index: 0, dataIndex: 0 }, { index: 1, dataIndex: 1 }];
+    const expectedResult = [
+      { index: 0, dataIndex: 0 },
+      { index: 1, dataIndex: 1 },
+    ];
     const state = shallowWrapper.state();
```

Comment:
Sorry my idea formatted this

Reply:
If you run `npm run prettier`, it will format everything for you.